pkg/logs/internal/launchers/file/launcher.go

```diff
@@ -342,10 +343,12 @@ func (s *Launcher) createRotatedTailer(t *tailer.Tailer, file *tailer.File, patt
 }
 
 func CheckProcessTelemetry(stats *util.ProcessFileStats) {
+	AgentOpenFilesStr := strconv.FormatFloat(stats.AgentOpenFiles,'f',0,64)
+	OsFileLimitStr := strconv.FormatFloat(stats.OsFileLimit,'f',0,64)
 	if stats.AgentOpenFiles/stats.OsFileLimit > 0.9 {
-		log.Warnf("Agent process is close to OS file limit of %v. Agent process currently has %v files open.", stats.OsFileLimit, stats.AgentOpenFiles)
+		log.Warnf("Agent process is close to OS file limit of %v. Agent process currently has %v files open.", OsFileLimitStr, AgentOpenFilesStr)
```

Comment:
Warnf can format floats too:

```suggestion
		log.Warnf("Agent process is close to OS file limit of %.0f. Agent process currently has %.0f files open.", stats.OsFileLimit, stats.AgentOpenFiles)
```

Reply:
Turns out I can keep `%v` now that I've switched from `float64` to `uint64`.